### DISCUSSION TITLE
Sample app that builds and makes a simple http call

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#242] [Show and tell]
here you go
[gary-roku-tv-skill.zip](https://github.com/OpenVoiceOS/ovos-buildroot/files/14476311/gary-roku-tv-skill.zip)

**Comment by mushie4282** [comment]
Thanks!

**Reply by geetee24** [reply]
base on this sample.

https://github.com/OpenVoiceOS/skill-ovos-hello-world/tree/dev

**Comment by mikejgray** [comment]
Nice sample! A few suggestions:

1. create_skill is no longer required under OVOS, so you can completely remove that function
2. I'd recommend switching to [ovos-lingua-franca](https://github.com/OpenVoiceOS/ovos-lingua-franca) instead of lingua-franca, as the latter hasn't been updated for a long time and frequently causes dependency conflicts in OVOS
3. Your __init__ method isn't passing `*args, **kwargs`, which will eventually cause load issues in OVOS. You'll want something like `def __init__( self, *args, **kwargs )`, which should also be done in your OVOSSkill call on the next line of code

**Comment by geetee24** [comment]
My sample came from The official Ovos sample

**Comment by mikejgray** [comment]
https://github.com/OpenVoiceOS/skill-ovos-hello-world/blob/42bee014eebbd251e874a1fdb4e9766086e83c18/__init__.py#L15
https://github.com/OpenVoiceOS/skill-ovos-hello-world/blob/42bee014eebbd251e874a1fdb4e9766086e83c18/__init__.py#L20

The official sample does not import lingua-franca, nor does it have a `create_skill()` function

**Reply by mikejgray** [reply]
It also appears that you haven't included lingua-franca in the requirements.txt, so this example will fail to load once it's been installed

**Comment by mushie4282** [comment]
Where is handle_tv_control_jump called? Is it in your mycroft.conf or a different file?

**Comment by geetee24** [comment]
Different file. But it just makes a similar HTTP call.

**Reply by mushie4282** [reply]
Were you able to confirm whether the bus emitted a message after you said your set command for this skill? I'm having trouble connecting my emitted message to my skill.

**Comment by geetee24** [comment]
my sample doesnt emit a custom message. it just makes http call
